src/main/scala/de/heikoseeberger/sbtfresh/Template.scala

```diff
@@ -349,22 +226,7 @@ private object Template {
   def `package`(packageSegments: Vector[String], author: String): String = {
     val superPackage = packageSegments.init.mkString(".")
     val lastSegment = packageSegments.last
-    s"""|/*
```

Comment:
Header part has been removed from Template because sbt-header anyway will add it to final project at first compilation.